src/shops/api/shops/__fixtures__/hornbach/category-page1.html

```
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html><body>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)